.circleci/config.yml

```diff
@@ -445,26 +444,17 @@ jobs:
               | sed -n 's:^ *BUILD_DIR = \(.*\)/Build/Products$:export DERIVED_DATA_PATH="\1":p' \
               >> $BASH_ENV
       - run:
-          name: Testing << parameters.scheme >> on AWS Device Farm
+          name: Testing << parameters.scheme >> on firebase
           command: |
-            make test-with-device-farm \
+            make test-with-firebase \
               SCHEME=<< parameters.scheme >> \
               APP_NAME=<< parameters.app-name >> \
-              CONFIGURATION=Release \
+              CONFIGURATION=Debug \
               BUILD_DIR="$DERIVED_DATA_PATH" \
               XCODE_WORKSPACE=Apps/Apps.xcworkspace
           when: always
-      # get xcresults here, may be zipped with more than one result
-      - run:
-          name: "Gathering results"
-          command: make gather-results BUILD_DIR="$DERIVED_DATA_PATH"
-          when: always
-      - run:
-          name: "Parsing xcresults for errors"
-          command: |
-            RESULTS=`find "$DERIVED_DATA_PATH/testruns" -name '*.xcresult'`
-            xargs swift run --package-path scripts/xcparty xcparty \<<< "$RESULTS" | tee "$DERIVED_DATA_PATH/testruns/failures.txt"
-          when: on_fail
```

Comment:
Did you prune this action because of missing `xcresult`? If so, we should recover it as soon as we add `result_bundle` to the `scan` action.

Reply:
The CI will give the output uri of firebase, we can check the failed result with this uri directly. This is the way we are using for Android too.

Reply:
@macdrevx Do we still need `xcparty` tool when working with Firebase?

Reply:
I think that tool was used to parse the results from device farm and fail the Circle CI job if it found test failures. I assume we'd want something similar with firebase.